packages/eslint-config-nori/package.json

```diff
@@ -30,18 +30,18 @@
     "react-dom": "~17"
   },
   "dependencies": {
-    "@types/eslint": "8.4.3",
     "@fintechstudios/eslint-plugin-chai-as-promised": "3.1.0",
-    "@graphql-eslint/eslint-plugin": "^3.19.1",
+    "@graphql-eslint/eslint-plugin": "^3.20.1",
     "@next/eslint-plugin-next": "^13.4.4",
+    "@types/eslint": "8.4.3",
     "@typescript-eslint/eslint-plugin": "^5.59.8",
     "@typescript-eslint/parser": "^5.59.8",
     "eslint": "^8.41.0",
     "eslint-config-airbnb": "19.0.4",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-next": "^13.4.4",
     "eslint-config-prettier": "^8.8.0",
-    "eslint-config-typescript": "^3.0.0",
```

Comment:
I removed this because:
  1. we didn't appear to be using this.
  2. the package hasn't been updated in several years